Makefile

```diff
@@ -1,12 +1,13 @@
 ARCHS=linux linux64 mac64 win32
-MANIFEST=manifest.json
+MANIFEST=extension/manifest.json
 
 VERSION=`grep -Eo '"version"\:\s*"[^"]+' $(MANIFEST) | grep -Eo '[0-9].*'`
 
 define build-xpis
+	cd extension; \
```

Comment:
I think we should restore the current directory after the for loop below?

pushd extension
for ...
  ...
done
popd

should work.